plugins/node/instrumentation-cucumber/package.json

```diff
@@ -63,7 +63,7 @@
     "sinon": "14.0.0",
     "test-all-versions": "5.0.1",
     "ts-mocha": "10.0.0",
-    "typescript": "4.3.5"
```

Comment:
Seem like 4.3.5 already support `Symbol Index Signatures`, the main branch have no issue with intrumentation-cucumber but fail with instrumentation-aws-sdk